core/README.md

```diff
@@ -45,7 +45,7 @@ Infrastructure interface includes:
 ### Special Modular
 * [Approver](./module/README.md) : Approver is the modular to handle ask approval request, 
   handles CreateBucketApproval and CreateObjectApproval.
-* [Authorizer](./module/README.md): Authorizer is the modular to authority verification.
+* [Authenticator](./module/README.md): Authenticator is the modular to authentication verification.
```

Comment:
Refine `authentication verification` to `verify authentication`?

Reply:
I don't do this because I did this in another pr to avoid conflicts. [pr](https://github.com/bnb-chain/greenfield-storage-provider/pull/544)